modules.yml

```diff
@@ -138,6 +92,21 @@ modules:
 
 # -- end faudio --
 
+# -- vkd3d --
+
+  - name: SPIRV-Headers
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: "https://github.com/KhronosGroup/SPIRV-Headers/archive/1.4.1.tar.gz"
+        sha256: a244f0629f75eb450e090cd773d30e22367cb231e964c7492588eb9000201fd1
```

Comment:
I thought these come through Sdk.

Reply:
Maybe the SDK should include these indeed, but there is no `/usr/include/spirv`.

Reply:
Looks like they might not be getting shipped. There's just this thing that you need to be really careful combining different projects for end result to be supported. Not all version combinations are okay. https://gitlab.com/freedesktop-sdk/freedesktop-sdk/blob/master/elements/components/spirv-tools.bst#L44